### ISSUE TITLE
Regression in Accessibility Audits

### ISSUE THREAD

**Issue by joelhsmith** [GoogleChrome/lighthouse#6167]
#### Provide the steps to reproduce
1. Run LH on http://cats.com

#### What is the current behavior?
"details" > "items" > "explanation" is missing.

```
    "color-contrast": {
      "id": "color-contrast",
      "title": "Background and foreground colors do not have a sufficient contrast ratio.",
      "description": "Low-contrast text is difficult or impossible for many users to read. [Learn more](https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse).",
      "score": 0,
      "scoreDisplayMode": "binary",
      "rawValue": false,
      "details": {
        "type": "table",
        "headings": [{
          "key": "node",
          "itemType": "node",
          "text": "Failing Elements"
        }],
        "items": [{
          "node": {
            "type": "node",
            "selector": "#notify_me",
            "path": "1,HTML,1,BODY,1,SECTION,0,DIV,1,DIV,0,FORM,2,BUTTON",
            "snippet": "<button id=\"notify_me\" type=\"submit\">Give Me 20% Off</button>",
            MISSING EXPLANATION
          }
        }],
        "impact": "serious",
        "tags": [
          "cat.color",
          "wcag2aa",
          "wcag143"
        ]
      }
    },
```

#### What is the expected behavior?
Explanation should be there.

```
            "explanation": "Fix any of the following:\n  Element has insufficient color contrast of 2.37 (foreground color: #ffffff, background color: #ff847d, font size: 16.5pt, font weight: normal). Expected contrast ratio of 4.5:1"
```

These error messages are not repetitive. Most of them contain custom explanations with the help text to diagnose the specific failing element and how to fix it.  The misconception that they were all just duplicates almost led to its removal before, but I explained the situation and saved them from deletion:

**Related issues**
I had made the case to keep those audits here and they were kept:
https://github.com/GoogleChrome/lighthouse/issues/5402

#### Environment Information
* Affected Channels: 
* Lighthouse version: 
* Node.js version: 8.9.1
* Operating System: OSX 10.12.6

**Comment by joelhsmith** [comment]
Sorry was testing on old version. My bad.
